src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -3835,23 +3835,28 @@ private SyntaxToken TryEatCheckedOrHandleUnchecked(ref SyntaxToken operatorKeywo
             Debug.Assert(IsStartOfPropertyBody(this.CurrentToken.Kind));
 
             var accessorList = this.CurrentToken.Kind == SyntaxKind.OpenBraceToken
-                ? this.ParseAccessorList(isEvent: false)
+                ? this.ParseAccessorList(AccessorDeclaringKind.Property)
                 : null;
 
             ArrowExpressionClauseSyntax expressionBody = null;
             EqualsValueClauseSyntax initializer = null;
 
-            // Check for expression body
-            if (this.CurrentToken.Kind == SyntaxKind.EqualsGreaterThanToken)
-            {
-                expressionBody = this.ParseArrowExpressionClause();
-            }
-            // Check if we have an initializer
-            else if (this.CurrentToken.Kind == SyntaxKind.EqualsToken)
+            // PROTOTYPE: SyntaxKind.GetAccessorDeclaration does seem correct for an initializer.
+            // PROTOTYPE: Confirm that field should be a keyword, and bind to the backing field, in a property initializer: object P { get; } = field;
```

Comment:
this is interesting case for sure.  i actually have no idea what should happen here.  Property intitializers can't actually reference `this` or fields.  But if we do *not* have this bind to the `field` keyword, this could then bind to somethgin higher... and i have no idea if we want that :)

Reply:
Removed this case, assuming we will decide `field` is not a keyword (and should not bind to the backing field) in the property initializer.